src/main/java/org/zalando/planb/provider/CassandraConfig.java

```diff
@@ -19,6 +20,7 @@ Session initializeCassandra() {
         final Cluster.Builder builder = Cluster.builder();
 
         builder.addContactPoints(cassandra.getContactPoints().split(","));
+        builder.withAddressTranslator(new EC2MultiRegionAddressTranslator());
```

Comment:
I can assist with the config stuff, if you want. we could for example put the class name of the address translator into the application.yaml file and create an instance of it in CassandraProperties.